django/contrib/auth/forms.py

```diff
@@ -197,6 +197,9 @@ class BaseUserCreationForm(SetPasswordMixin, forms.ModelForm):
     """
     A form that creates a user, with no privileges, from the given username and
     password.
+
+    This is the documented base class for customizing the user creation form.
+    It should be kept mostly unchanged to ensure consistency and compatibility.
```

Comment:
Despite the new test (thanks Sarah!), I think that also adding a small note in the docstring will help future readers and contributors.